### DISCUSSION TITLE
[Form] How to have rows with fields aligned?

### DISCUSSION THREAD

**Discussion by smartm0use** [fomantic/Fomantic-UI#1616] [Collections (form, grid, menu, message, table, ...)] [answered]
Hello,
I would have a form like this:

![download](https://user-images.githubusercontent.com/5399265/88793817-baaa1200-d19d-11ea-81a6-7e3980ebc8a0.png)

How to have the first field of second row aligned?
I cannot find any class to do that in the documentation.

Thank you in advance!

**Comment by ko2in** [comment]
What do you mean by aligned of the first field? Your question is not clear enough.

**Comment by lubber-de** [comment, marked as answer]
@smartm0use  Two options:

1. Use `n wide field` to define the width individually.
See https://jsfiddle.net/lubber/p17zt3qf/2/

2. Use an empty field
See https://jsfiddle.net/lubber/p17zt3qf/4/
This has the advantage of staying responsive

**Comment by smartm0use** [comment]
Thank you, that's what I'm looking for :)
